tools/dataset_fixtures.py

```diff
@@ -146,7 +146,7 @@ def get_start_end_date(date_string):
             'model': 'datasets.datafile',
             'pk': r['id'],
             'fields': {
-                'title': r.get('description', '')[0:64],
+                'name': r.get('description', '')[0:64],
```

Comment:
@rossjones 

Is this key name change necessary because of line 25 on [this file](https://github.com/datagovuk/publish_data_alpha/blob/master/src/datasets/migrations/0001_initial.py)?